vuepress/docs/next/docs/consume/identity-management.md

```diff
@@ -1,53 +1,66 @@
 
-### Core
-Authorization for the entando-core and WCMS is provided by the Entando user management database. When a user is authenticated to the entando-core  a copy of that user is added to the Entando user management database.  That copy is made in support of the authorization flow noted below.
+Below are the steps to grant a user one or more roles for a specific client. This controls permissions when configuring the microservice. Note: when a microservice is installed in Entando, a corresponding client (and set of roles) is created per its plugin definition.
+
+1. [Login to your Keycloak instance](#logging-into-your-keycloak-instance) 
+2. Select `Users` from the menu on the left
+3. Use the search box to find the appropriate user, e.g. "admin"
+4. Click on the user ID
+
+![find-admin.png](./img/find-admin.png)
+
+5. Click on the `Role Mappings` tab
+6. Use the `Client Roles` drop-down menu to specify the microservice client
+7. Select from the client's `Available Roles`
 
-As noted above when a user is authenticated to the entando-core via keycloak a copy of that user is added to the entando-core user management database in support of WCMS functionality.  Using the App Builder WCMS roles and groups can be assigned to a user for access to functions in the App Builder or for portal-ui based content access in the runtime application The code that copies the user into the entando-core can be customized to automatically create groups and roles as needed for an application but it is something that must be done on a per implementation basis.
+![find-roles.png](./img/find-roles.png)
 
-For more details on the code that copies users and data to the WCMS database see the [entando-keycloak-plugin](https://github.com/entando/entando-keycloak-plugin). The readme in that project includes properties that are available to your Entando app.
+8. Use the `Add Selected` button to move the desired roles to `Assigned Roles`. These will subsequently appear under `Effective Roles`.
 
-For a deeper look See: [KeycloakAuthorizationManager.java](https://github.com/entando/entando-keycloak-plugin/blob/master/src/main/java/org/entando/entando/keycloak/services/KeycloakAuthorizationManager.java) in the plugin for an example of adding attributes programatically. In particular, the [processNewUser](https://github.com/entando/entando-keycloak-plugin/blob/master/src/main/java/org/entando/entando/keycloak/services/KeycloakAuthorizationManager.java#L43) method.
+![assign-roles.png](./img/assign-roles.png)
+### Core
+When a user is authenticated to the `entando-core` via Keycloak, a copy of that user is added to the `entando-core` user management database to enable WCMS functionality. Within the App Builder, WCMS roles and groups can be assigned to a user for access to App Builder functions or `portal-ui` content in the runtime application.
```

Comment:
"copy of that user" >> should it say copy of that user information or something like that?

"runtime application" >> is this referring to specific types of programs, I think called runtime programs, or is it talking about the execution of an application?

Reply:
no, copy of the user -> copy of the entity. similar to "copy the file" and not "copy the file's information"

leaving the runtime language as-is; just from a google search "runtime application" seems to be used frequently and sometimes interchangeably with runtime program. per one definition:
"A runtime program is a specific version of an application that is distributed for limited use. These kinds of software releases may be called runtime programs because they offer a runtime environment without other features that allow for more permanent use, such as building executable files."
given this def application seems more appropriate than program ?? i suspect there are nuances... @nshaw

Reply:
1. Copy of the user is good enough. It's really only a couple fields but functionally that's how it behaves.
2. Yes, application is better. It's somewhat interchangeable but we build web applications not web programs. Programs just sounds dated at this point for some reason.

Reply:
I must have worded this badly. I wasn't suggesting we change application to program. It was unclear whether you were talking about the application during the runtime or something else.